GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -224,6 +224,14 @@ private IReadOnlyList<MenuCommand> CreateViewMenuCommands()
                     ExecuteAction = () => _revisionGrid.ToggleShowReflogReferences(),
                     IsCheckedFunc = () => AppSettings.ShowReflogReferences
                 },
+                new MenuCommand
+                {
+                    Name = "ShowStashReference",
+                    Text = "Show stash reference",
```

Comment:
I still want to change this...

Reply:
I'm not against changing it, but. To what?

Reply:
https://github.com/gitextensions/gitextensions/pull/9555#discussion_r701216184

I suggest `Show latest stash` after reading the man page again.
@RussKie as a non user of stash was too quick to suggest "stash reference".

Then we are done with this...

Reply:
...then we should also rename all other things, like settings name, property names, etc.

Reply:
It is changed to ```Show latest stash```.